src/Model/PickupPoint.php

```diff
@@ -75,52 +75,52 @@ public function setCode(PickupPointCode $code): void
         $this->code = $code;
     }
 
-    public function getName(): string
+    public function getName(): ?string
```

Comment:
Why did all of these became nullable?

Reply:
Because they can be null in fact. All that variables was not initialized with some default values, so they should be nullable

Reply:
The only nullable property (in the constructor) is the latitude and longitude

Reply:
Yeah, missed that. Thanks
Reverted